README.md

```diff
@@ -140,6 +140,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 * [indiscipline/zrtstr](https://github.com/indiscipline/zrtstr) — A command line utility for checking if stereo wav files are faux-stereo (i.e. have identical channels) and converting such files to mono. [<img src="https://api.travis-ci.org/indiscipline/zrtstr.svg?branch=master">](https://travis-ci.org/indiscipline/zrtstr)
 * [Polaris](https://github.com/agersant/polaris) — A music streaming application.  [<img src="https://api.travis-ci.org/agersant/polaris.svg?branch=master">](https://travis-ci.org/agersant/polaris)
 * [Spotify TUI](https://github.com/Rigellute/spotify-tui) — A Spotify client for the terminal written in Rust. [<img src="https://github.com/Rigellute/spotify-tui/workflows/Continuous%20Integration/badge.svg">](https://github.com/Rigellute/spotify-tui/actions?workflow=Continuous+Integration)
+* [Spotifyd](https://github.com/Spotifyd/spotifyd) — An open source Spotify client running as a UNIX daemon. [<img src="https://github.com/Spotifyd/spotifyd/workflows/Continuous%20Integration/badge.svg">](https://github.com/Spotifyd/spotifyd/actions?workflow=Continuous+Integration)
```

Comment:
The badge doesn't specify a branch. Can you please specify the main build branch (master in this case)

Reply:
Thanks for adding the branch to Spotifyd but you appear to have removed the branch from Spotify TUI. Could you put it back please?

Reply:
I did not remove the branch from the Spotify TUI. It was originally not there, but I can add it.